src/ex3/fs.js

```diff
@@ -0,0 +1,21 @@
+import fs from "fs/promises";
+
+export async function writeToFile(item, addNewLine, flag) {
+  try {
+    const newLine = addNewLine ? "\n" : "";
```

Comment:
I think it would make more sense to just add new line where you want it in the string you send here to the write function, it's a little weird here. And if flag is option which I see it is then worth giving it a default value or putting it in an options = {} object so that by reading the function it's clear that it's optional